src/wallet/wallet2.cpp

```diff
@@ -4867,27 +4876,16 @@ bool wallet2::sign_tx(unsigned_tx_set &exported_txs, const std::string &signed_f
   return true;
 }
 //----------------------------------------------------------------------------------------------------
-bool wallet2::load_tx(const std::string &signed_filename, std::vector<tools::wallet2::pending_tx> &ptx, std::function<bool(const signed_tx_set&)> accept_func)
+bool wallet2::load_tx_from_str(const std::string &tx_s, std::vector<tools::wallet2::pending_tx> &ptx, std::function<bool(const signed_tx_set&)> accept_func)
```

Comment:
Maybe simply use `s` instead of `tx_s`? The variable name looks quite short, but the intent should be clear from the context.